test/contact-summary.spec.js

```diff
@@ -38,7 +38,11 @@ describe('getContactSummary', () => {
     const reports = [];
     const lineage = [];
     await harness.getContactSummary(contact, reports, lineage);
-    expect(functionStub.args[0]).to.deep.eq([contact, reports, lineage]);
+    const slicedArgs = functionStub.args[0].slice(0, 3);
```

Comment:
Can you use `to.deep.include` instead of `to.deep.eq` if you just want to validate a subset?